Source/Libraries/FaultData/DataReaders/EMAXReader.cs

```diff
@@ -114,8 +115,19 @@ public void ExportToCOMTRADE()
 
                     comtradeAnalog.Multiplier = (max - min) / (2 * short.MaxValue);
                     comtradeAnalog.Adder = (max + min) / 2.0D;
-                    comtradeAnalog.PrimaryRatio = double.TryParse(analogChannel.primary, out num) ? num * unitMultiplier : 0.0D;
-                    comtradeAnalog.SecondaryRatio = double.TryParse(analogChannel.secondary, out num) ? num * unitMultiplier : 0.0D;
+                    // Setting Secondary to 1 and adjusting primary accordingly
```

Comment:
Why are we doing this? If it's actually necessary to do this, this comment would be a great place to explain why.

Reply:
Honestly I am not too sure myself, something TVA asked for. I think it's for human readability. @elwills do you know why?

Reply:
I don't know the why. That would be a question for @clackner-gpa

Reply:
Human readability mostly